src/main/java/io/github/railroad/utility/FileUtils.java

```diff
@@ -1,48 +1,38 @@
 package io.github.railroad.utility;
 
+import javafx.stage.FileChooser;
+import org.jetbrains.annotations.NotNull;
+
 import java.io.File;
 import java.io.FileWriter;
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.List;
-import java.util.Objects;
-import java.util.Optional;
-
-import org.jetbrains.annotations.NotNull;
-
-import javafx.stage.FileChooser;
+import java.util.*;
 
 public class FileUtils {
 
-	public static File createFolder(String pathIn) {
+	public static File createFolder(String pathIn) { // Unused Method
 		File newFolder = new File(pathIn);
-		newFolder.mkdir();
+		newFolder.mkdir(); // Return is ignored. Add check.
 		return newFolder;
 	}
 
-	public static void deleteFile(File file) {
```

Comment:
Why did you remove this?

Reply:
The method doesn't *do* anything. It's not even a shorthand. You can just call what it calls, since you pass in `file` anyway.

Reply:
i mean thats fair enough. but in the future, this will be added back. because this method wont just do this, this was just a template. it will also need to update things

Reply:
Sounds good. Probably should include a JavaDoc that says that :wink: